scripts/provision.sh

```diff
@@ -275,3 +277,21 @@ if [ "$ARM" = "1" ]; then
 	      export PATH=$PATH:`pwd`/$EXPECTEDARMGCCFILENAME/bin
     fi
 fi
+
+#--------------------------------------------------------------------------------
+EMSCRIPTEN_VERSION="3.1.54"
```

Comment:
I am not sure if this is the correct place to configure the emscripten version

Reply:
I think this is fine - it's where we have the ARM compiler version listed as well